noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1968,10 +2144,15 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST21">
         <title>Disposal count</title>
+        <description>
+            Provides information about the number of Disposals related to Series
```

Comment:
`Provides information about the number of disposals of series and document descriptions, and tests whether the corresponding number in arkivuttrekk.xml (omfatterDokumenterSomErKassert) is correct`